CONTRIBUTING.md

```diff
@@ -163,7 +163,77 @@ in general try to follow them.
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `bug`                | Something that is advertised or intended to work isn't working as expected.                                                                                                                             |
+| `enhancement`        | Something that isn't an advertised feature that would be useful to users or maintainers.                                                                                                                |
+| `flaky test`         | A test unexpectedly failed during CI, showing that there is a problem with the tests or test setup that is causing the tests to intermittently fail.                                                    |
+| `good first issue`   | Implementing this issue would not require specialized or in-depth knowledge about the component and is ideal for a new or first-time contributor to take.                                               |
+| `help wanted`        | The code owners for this component do not expect to have time to work on it soon, and would welcome help from contributors.                                                                             |
+| `needs discussion`   | This issue needs more input from the maintainers or community before work can be started.                                                                                                               |
+| `needs triage`       | This label is added automatically, and can be removed when a triager or code owner deems that an issue is either ready for work or should not need any work.                                            |
+| `waiting for author` | Can be applied when input is required from the author before the issue can move any further.                                                                                                            |
+| `priority:p0`        | A critical security vulnerability or Collector panic using a default or common configuration unrelated to a specific component.                                                                         |
+| `priority:p1`        | An urgent issue that should be worked on quickly, before most other issues.                                                                                                                             |
+| `priority:p2`        | A standard bug or enhancement.                                                                                                                                                                          |
+| `priority:p3`        | A technical improvement, insignificant bug, or other minor issue. Generally something that is considered a "nice to have."                                                                              |
```

Comment:
a bug is rarely insignificant, especially to the person who spent the time reporting it. How about changing this to "low-priority bug"?

Reply:
Agreed. I wanted to indicate that the bug probably had a workaround (otherwise it would be a p2), but I'm not sure we should be that prescriptive. Amended.